docs/configuration.rst

```diff
@@ -260,6 +260,7 @@ It can be set to:
 
 * ``http`` to accept the ``PROXY`` protocol on nginx's HTTP proxy ports
 * ``mail`` to accept the ``PROXY`` protocol on nginx's mail proxy ports
+* ``sieve`` to accept the ``PROXY`` protocol on nginx's managesieve proxy port
```

Comment:
Shall I suggest merging that with `mail` ? Having two made configuration easy (all combinations from "none, only mail, only http, both" are configurable).

With a third option, you either need 7 option names (to allow "mail + sieve" or "http + sieve", or complex string comparison.

Reply:
Ok, changed in 801e6fe0